src/coreclr/jit/emitarm64.cpp

```diff
@@ -5590,10 +5590,23 @@ void emitter::emitIns_R_R_I(
             isLdSt     = true;
             break;
 
+        case INS_st2:
```

Comment:
could you possibly combine this with the `case INS_ld1, INS_st1` below?

Reply:
Done